SIPS/sip-7.md

```diff
@@ -65,12 +65,20 @@ This SIP describes a couple of components for the initial version of the UI:
 4. A button component, rendering the text as a HTML button (`button`), with an optional ID for handling user input.
     1. The button can have two variants: `primary` and `secondary`.
     2. If the variant is not specified, `primary` SHOULD be used.
+    3. The button can have two types: `button` and `submit`.
+    4. If the type is not specified, `button` SHOULD be used.
 5. A divider component, rendering a HTML thematic break (`hr`).
 6. A spacer component, pushing content around it to the top and bottom, similar to how `flex-grow` works.
 7. A spinner component, rendering a spinner to indicate loading data.
+8. A copyable component, rendering the value along a copy icon that copies the value in the clipboard.
+9. A Form component, which renders a list of `children` components and define a form, with an ID for handling user input.
+10. An input component, rendering an HTML input (`input`) with an ID for handling user input.
+  1. The input can be of 4 different types: `text`, `password`, `number`, `search`
```

Comment:
What's the use case for `search` inputs?

Reply:
This is one of the available input types in the extension. I just allowed all the input types available on the `FormTextField` component that we use

Reply:
But it won't be handled any different compared to regular text inputs, right? I'd say we can just remove it.